.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,64 @@
+name: CI/CD
+
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+
+env:
+    CI: true
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Use Node.js 18
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
+
+            - run: npm install
+
+            - run: npm run build
+
+            - run: npm run lint
+
+    release:
+        runs-on: ubuntu-latest
+        needs:
+            - test
+        if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/beta'
```

Comment:
This will need to be `master`. You can probably drop the `beta` condition.

Reply:
Done